### PR TITLE
Frontend: Change team list to collapsible card layout to show extra details

### PR DESCRIPTION
**Pull request by spyshiv**
Fix for #371 

**Updated view:** 
`My Host Team`
`My Participant Team` 

**Screenshot:**

screenshot from 2017-03-30 19 50 04

@deshraj @aka-jain : Please review.

### CODE REVIEW

File: frontend/src/views/web/challenge-host-teams.html [hunk 1]
```diff
@@ -15,35 +15,42 @@
...
+                                    <a ng-click="challengeHostTeams.confirmDelete($event, value.id)"><i class="fa fa-trash list-icon text-highlight w-300 right grey-text"></i></a>
+                                    <a ng-click="challengeHostTeams.inviteOthers($event, value.id)"><i class="fa fa-user-plus list-icon text-highlight w-300 right"></i></a>
+                                    <br>
+                                    <span><b class="text-med-black">Created by: </b>{{value.created_by}}</span>
                                 </div>
+                                <div class="clearfix"></div>
                             </div>
-                        </section>
+                            <div class="collapsible-body">
+                                <h6><b>Participants:</b></h6>
```
Comment (deshraj) [inline frontend/src/views/web/challenge-host-teams.html]: Can you please replace `Participants` with `Team Members`

File: frontend/src/views/web/teams.html [hunk 2]
```diff
@@ -15,40 +15,37 @@
...
-                                            <button class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14" type="submit" ng-click="teams.selectExistTeam()" value="Submit">Participate</button>
-                                            <br>
-                                            <br>
-                                            <span class="text-dark-black"><strong >{{teams.existTeamError}}</strong></span>
-                                        </div>
-                                    </div> -->
+                                <div class="clearfix"></div>
+                            </div>
+                            <div class="collapsible-body">
+                                <h6><b>Participants:</b></h6>
```
Comment (deshraj) [inline frontend/src/views/web/teams.html]: Same as above comment. :)

File: frontend/src/views/web/challenge-host-teams.html (hunk 1 above)
Comment (spyshiv) [inline frontend/src/views/web/challenge-host-teams.html]: yeah sure..

**Comment by spyshiv** [comment]
Done.

**Pull request merged**